Libraries/BatchedBridge/MessageQueue.js

```diff
@@ -278,14 +277,8 @@ class MessageQueue {
       }
     }
 
-    /* $FlowFixMe(>=0.38.0 site=react_native_fb,react_native_oss) - Flow error
-     * detected during the deployment of v0.38.0. To see the error, remove this
-     * comment and run flow */
-    this._callbacks[cbID & ~1] = null;
-    /* $FlowFixMe(>=0.38.0 site=react_native_fb,react_native_oss) - Flow error
-     * detected during the deployment of v0.38.0. To see the error, remove this
-     * comment and run flow */
-    this._callbacks[cbID |  1] = null;
+    this._successCallbacks[callID] = this._failureCallbacks[callID] = null;
+
     // $FlowIssue(>=0.35.0) #14551610
     callback.apply(null, args);
```

Comment:
I think I didn't touch this because that seemed too obvious - does #14551610 say anything? (I guess that's internal). My guess is that since we'd want to throw an error if callback isn't set anyway, it's not worth the (small) performance hit to check it - noting that we do check in dev?

Cheers for having a look.